packages/orchestrator/internal/sandbox/checks.go

```diff
@@ -62,22 +79,20 @@ func (c *Checks) Start() {
 func (c *Checks) Stop() {
 	c.ctx.Lock()
 	c.ctx.Cancel()
+
+	if c.unregisterMetrics != nil {
+		if err := c.unregisterMetrics(); err != nil {
+			sbxlogger.I(c.sandbox).Error("failed to unregister metrics", zap.Error(err))
+		}
+	}
+
 	c.ctx.Unlock()
 }
 
 func (c *Checks) LogMetrics(ctx context.Context) {
 	logger := c.sandbox
 
-	if c.useLokiMetrics == "true" {
-		logger.LogMetricsLoki(ctx)
-	}
-
-	if c.useClickhouseMetrics == "true" {
-		logger.LogMetricsClickhouse(ctx)
-	}
-
-	// ensure backward compatibility if neither are set
-	if c.useClickhouseMetrics != "true" && c.useLokiMetrics != "true" {
+	if c.useLokiMetrics {
```

Comment:
removed the fallback, it's handled via env variables and it doesn't make sense to use loki if you explicitly pass `false`